packages/framework/src/Resources/config/parameters_common.yml

```diff
@@ -13,3 +13,5 @@ parameters:
 
     # Default extended classes mapping
     shopsys.entity_extension.map: {}
+
+    env(IGNORE_DEFAULT_ADMIN_PASSWORD_CHECK): 0
```

Comment:
Just question. Why is there 0/1 instead of false/true?

Reply:
There is no reason for that.

Reply:
Changed to `'0'` in #1386 for consistency with env vars in general (they always contain strings)